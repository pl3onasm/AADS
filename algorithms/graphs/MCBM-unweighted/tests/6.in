3 5

Sam Catcher_In_The_Rye 2
Sam Of_Mice_And_Men 1
Jonah Catcher_In_The_Rye 3
Jonah Of_Mice_And_Men 1
Jonah Lord_Of_The_Flies 2
Esther The_Great_Gatsby 1
Esther Catcher_In_The_Rye 4
Esther Of_Mice_And_Men 2
Esther Picture_Of_Dorian_Gray 1
Elijah Of_Mice_And_Men 3
Kate Of_Mice_And_Men 2
Kate Picture_Of_Dorian_Gray 1
Kate Lord_Of_The_Flies 2
