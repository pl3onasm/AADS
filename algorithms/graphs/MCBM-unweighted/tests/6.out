
--------------------
 Network
 Vertices: 10
 Edges: 13
--------------------
  Catcher_In_The_Rye[0]
  Elijah[1]: Of_Mice_And_Men(3)
  Esther[4]: Picture_Of_Dorian_Gray(1), Of_Mice_And_Men(2), Catcher_In_The_Rye(4), The_Great_Gatsby(1)  
  Jonah[3]: Lord_Of_The_Flies(2), Of_Mice_And_Men(1), Catcher_In_The_Rye(3)
  Kate[3]: Lord_Of_The_Flies(2), Picture_Of_Dorian_Gray(1), Of_Mice_And_Men(2)
  Lord_Of_The_Flies[0]
  Of_Mice_And_Men[0]
  Picture_Of_Dorian_Gray[0]
  Sam[2]: Of_Mice_And_Men(1), Catcher_In_The_Rye(2)
  The_Great_Gatsby[0]
--------------------

---------------
 Matching
---------------
  Kate -- Lord_Of_The_Flies
  Kate -- Picture_Of_Dorian_Gray
  Sam -- Of_Mice_And_Men
  Sam -- Catcher_In_The_Rye
  Esther -- Picture_Of_Dorian_Gray
  Esther -- Catcher_In_The_Rye
  Elijah -- Of_Mice_And_Men
  Jonah -- Lord_Of_The_Flies
  Jonah -- Of_Mice_And_Men
---------------
